Analysis/Tasks/PWGHF/taskXic.cxx

```diff
@@ -150,7 +150,7 @@ struct TaskXicMC {
         //Printf("MC Gen.: Y rejection: %g", RecoDecay::Y(array{particle.px(), particle.py(), particle.pz()}, RecoDecay::getMassPDG(particle.pdgCode())));
         continue;
       }
-      if (std::abs(particle.flagMCMatchGen()) == XicToPKPi) {
+      if (std::abs(particle.flagMCMatchGen()) == 1 << XicToPKPi) {
```

Comment:
This should be the first check.
Rapidity cut should be applied only for matched particles.
Have a look at `taskLc.cxx`.

Reply:
The rapidity check is now uniformed to the Lc task